apps/client/src/features/operator/Operator.tsx

```diff
@@ -114,9 +114,10 @@ export default function Operator() {
   // get fields which the user subscribed to
   const shouldEdit = searchParams.get('shouldEdit');
 
+  // we use '-' to separe multiple fields
+  // the fields may contain '_' if there are spaces in the name
```

Comment:
This comment mentions an `_` but the following code splits on `-` instead of the old `_` split